### PR TITLE
Add formatted logs 

### PR DESCRIPTION
**Pull request by calvinnhieu**
* Create static Logging class to configure logger
* Update log output in driver, mapper, reducer, worker

Example formatted log:
```
2018-10-23T02:06:33-0700 DEBUG matrix.lambdas.daemons.driver REQUEST_ID:382c11fa-af77-4cd0-8ffd-105c1460bbbe Driver running with parameters: bundle_fqids=['id1.version', 'id2.version'], format=test_format, bundles_per_worker=100
```

### CODE REVIEW

File: matrix/common/logging.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import logging
+import os
+import sys
+
+
+class Logging:
+    @staticmethod
+    def get_logger(name: str):
```
Comment (calvinnhieu) [inline matrix/common/logging.py]: Taken from upload service

File: matrix/common/logging.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+                                      datefmt="%Y-%m-%dT%H:%M:%S%z")
+        ch.setFormatter(formatter)
+        logger = logging.getLogger(name)
+        logger.handlers = []
+        logger.addHandler(ch)
+        logger.setLevel(logging.DEBUG)
+        return logger
+
+    @staticmethod
+    def set_correlation_id(logger: logging.Logger, id_name: str="REQUEST_ID", value: str=None):
```
Comment (calvinnhieu) [inline matrix/common/logging.py]: Taken from upload service

File: matrix/common/logging.py [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+        formatter = logging.Formatter('%(asctime)s %(levelname)s %(name)s' +
+                                      f' {id_name}:{value} %(message)s', datefmt="%Y-%m-%dT%H:%M:%S%z")
```
Comment (calvinnhieu) [inline matrix/common/logging.py]: Log format

File: matrix/common/logging.py [hunk 4]
```diff
@@ -0,0 +1,27 @@
+import logging
+import os
+import sys
+
+
+class Logging:
+    @staticmethod
+    def get_logger(name: str):
+        ch = logging.StreamHandler(sys.stdout)
+        log_level_name = os.environ['LOG_LEVEL'] if 'LOG_LEVEL' in os.environ else 'DEBUG'
```
Comment (mckinsel) [inline matrix/common/logging.py]: Maybe `os.environ.get("LOG_LEVEL", "DEBUG")`

File: matrix/lambdas/daemons/driver.py [hunk 5]
```diff
@@ -3,43 +3,55 @@
...
-            """
-            iter_args = [iter(bundle_fqids)] * self._bundles_per_worker
-            for bundle_fqid_group in itertools.zip_longest(*iter_args):
-                yield list(filter(None, bundle_fqid_group))
-
-        for bundle_fqid_group in _group_bundles(bundle_fqids, self._bundles_per_worker):
+        logger.debug(f"Driver running with parameters: bundle_fqids={bundle_fqids}, "
+                     f"format={format}, bundles_per_worker={self.bundles_per_worker}")
+
+        num_expected_mappers = int(math.ceil(len(bundle_fqids) / self.bundles_per_worker))
```
Comment (mckinsel) [inline matrix/lambdas/daemons/driver.py]: I think `math.ceil` returns an `int` anyway.

Comment (calvinnhieu) [reply matrix/lambdas/daemons/driver.py]: Ooh it returns a `float` in Python 2. Removing

**Comment by calvinnhieu** [comment]
Good idea! I'll add it to Parth and I's running list of future items heh:

https://docs.google.com/document/d/1XSZp_V2K-_nnjgh2gJatIqmixk1hpN23xmDPjB6eaDQ/edit

**Pull request merged**